pages/moderation.vue

```diff
@@ -12,11 +12,11 @@
 
 <script setup lang="ts">
 import { useModerationScreenStore, ModerationScreen } from "~/stores/moderationScreenStore"
-import { useRoute } from 'vue-router'
+// import { useRoute } from 'vue-router'
```

Comment:
I think this code can be removed?

Reply:
If you remove this code make sure all the other cypress tests are passing first. This is what was keeping the layout from shifting due to the env variable being implemented in development. Since the page layout is different for every other route once the env variable is introduced it can mess up the layout of every other page. I do not know either how on production we will make it know that it is on the moderation page to show the correct layout.

Reply:
Assuming this code isn't being utilized in the place it was previously being implemented

Reply:
oh I was just referring to the commented out portion

Reply:
Actually, this file shouldn't change. Will undo them.